schemas/event.yml

```
@@ -99,45 +220,108 @@
       example: user-password-change
 
     - name: outcome
-      level: extended
+      level: core
       type: keyword
-      short: The outcome of the event.
+      short: The outcome of the event. (Categorization Field)
```

```suggestion
      short: The outcome of the event. The lowest categorization field in the hierarchy.
```

Wording question: "lowest" or "last"? "lowest" sounds less important, which is not the case here. I really do want to know if the event of category "authentication" had outcome "success" or "failure" :-)

---

I think we need a visualization to communicate these better. For instance,
```
event.kind:                   event                     alert
                          /     |    \                /      \
event.category:      process  file  network       malware    intrusion_detection
                       ...   /     \
event.type:              access     change
                           |      /    |    \
event.action:            read  create modify rename
                               /   \
event.outcome:            success   failure
```

---

I love the diagram.  I think that the use of "high" to "low"  (and similarly top to bottom) when talking about hierarchical structures is well known, and I don't think readers will associated lowest with "less important."